tests/unit/test_io_utils.py

```diff
@@ -30,10 +30,16 @@ def test_assert_path_is_not_empty_file_exists(self):
         with pytest.raises(FileExistsError):
             io_utils.assert_path_is_not_empty(
                 path="/fake/file/path", description="Io Utils"
-            ), "Provided a fake file path, function is fail"
+            ), "Provided path to an empty file, function expected to fail"
 
     def test_assert_path_is_not_empty(self, tmp_path):
         # Create temp file path
```

Comment:
The new edit has this comment out of place and needs to be moved right above the line that creates a temp file `empty_temp_file_path = tmp_path / "empty.text"`

Reply:
fixed